content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,310 @@ menu:
   default:
     identifier: automations
     parent: w-b-models
-title: Automations
+title: Create and manage automations
```

Comment:
I am totally in favor of action-based titles. Just a heads up, we use the product name because Lukas (CEO) raged quit my action-based titles (years ago). That's to say, you might get feedback to change this to "Automations"

Reply:
Reverted this change for now